src/runtime/client-hydrate.ts

```diff
@@ -254,7 +254,7 @@ const clientHydrate = (
   }
 };
 
-export const initializeDocumentHydrate = (node: d.RenderNode, orgLocNodes: Map<string, any>) => {
+export const initializeDocumentHydrate = (node: d.RenderNode, orgLocNodes: d.PlatformRuntime['$orgLocNodes$']) => {
```

Comment:
I think this introduces a new strictNullCheck error, since `$orgLocNodes$` is an optional field on `PlatformRuntime`. I think in the context of this function, it makes sense for us to narrow it if we want to use the typings on `PlatformRuntime`:
```diff
- export const initializeDocumentHydrate = (node: d.RenderNode, orgLocNodes: d.PlatformRuntime['$orgLocNodes$']) => {
+ export const initializeDocumentHydrate = (node: d.RenderNode, orgLocNodes: Exclude<d.PlatformRuntime['$orgLocNodes$'], undefined>) => {
```

Note: If we use this, we'll probably want to format it with Prettier:
```suggestion
export const initializeDocumentHydrate = (
  node: d.RenderNode,
  orgLocNodes: Exclude<d.PlatformRuntime['$orgLocNodes$'], undefined>,
) => {
```